src/cts/test/simple_test.tcl

```diff
@@ -10,4 +10,12 @@ set_wire_rc -clock -layer metal3
 clock_tree_synthesis -root_buf CLKBUF_X3 \
                      -buf_list CLKBUF_X3 \
                      -wire_unit 20 \
-                     -obstruction_aware    
+                     -obstruction_aware \
+                     -apply_ndr    
+
+write_def simple_test_out.def
+set fp [open simple_test_out.def]
+set fc [read $fp]
+close $fp
+puts $fc
+exec rm simple_test_out.def
```

Comment:
This should be handled by having a defok file - see balance_levels.tcl for example
```
set def_file [make_result_file balance_levels.def]
write_def $def_file
diff_files balance_levels.defok $def_file
```
No files should be written outside the results dir.

Reply:
Done.